web/templates/user/show.html.eex

```diff
@@ -1,31 +1,64 @@
-<h2>Show user</h2>
 
-<ul>
-  <li>
-    <strong>First Name:</strong>
-    <%= @user.name %>
-  </li>
-  <li>
-    <strong>Family Name:</strong>
-    <%= @user.family_name %>
-  </li>
-  <li>
-    <strong>Nickname:</strong>
-    <%= @user.nickname %>
-  </li>
-  <li>
-    <strong>Email:</strong>
-    <%= @user.email %>
-  </li>
-  <li>
-    <strong>Student ID:</strong>
-    <%= @user.student_id %>
-  </li>
-  <li>
-    <strong>Comments:</strong>
-    <%= @user.comments %>
-  </li>
-</ul>
+  <div class="row">
+    <div class="
+      col-xs-12
+      col-sm-offset-1 col-sm-10
+      col-md-offset-2 col-md-8
+      col-lg-offset-2 col-lg-8
+    ">
+      <div class="row middle-xs page-header">
+        <div class="col-xs-6 col-sm-9 col-md-10 page-title">
+          <%= CoursePlanner.SharedView.display_user_name(@user) %>
+        </div>
+        <%= if @conn.assigns.current_user.role == "user" do %>
```

Comment:
I changed that, that was an accident, I used the coordinator show page, and I replaced that one too. I will update the PR soon